src/components/TextListItem.tsx

```diff
@@ -42,10 +42,12 @@ export const TextListItem: NamedExoticComponent<Props> & {
 }>(({ navigation, item, noSubtitle, leftImage }) => {
   const { routeName, params, subtitle, title, bottomDivider, topDivider, picture } = item;
 
+  const titleText = <BoldText>{trimNewLines(title)}</BoldText>;
+
   return (
     <ListItem
-      title={noSubtitle || !subtitle ? undefined : <RegularText small>{subtitle}</RegularText>}
-      subtitle={<BoldText>{trimNewLines(title)}</BoldText>}
+      title={noSubtitle || !subtitle ? titleText : <RegularText small>{subtitle}</RegularText>}
+      subtitle={noSubtitle || !subtitle ? undefined : titleText}
```

Comment:
i do not get this fully right now. can you make a real sentence to help me when to show a `title` and when a `subtitle`?

Reply:
Previously we had always shown the `title`-prop as the subtitle, and the `subtitle`-prop as the title, if it was present and if the `noSubtitle`-flag was not set.

I now changed the logic to show the `title`-prop as the title instead of as the subtitle, if there is no `subtitle`-prop that we want to show (i.e. it is `undefined` or the `noSubtitle`-flag is set).

the condition for the ternary remains the same, and I added the same ternary condition to the `subtitle` of `ListItem` as well.

This way the `titleText` is always shown, and only as a subtitle if we want to show a subtitle as well.